src/test/java/com/example/java/maven/ticketVendingMachine/TicketPrinterTest.java

```diff
@@ -0,0 +1,49 @@
+package com.example.java.maven.ticketVendingMachine;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import org.junit.jupiter.api.BeforeEach;
+
+import static org.mockito.BDDMockito.then;
+import static org.mockito.Mockito.times;
+
+public class TicketPrinterTest {
+
+    private final static String PRINT_TICKETS = "Here are your tickets:";
+    MessagePrinter messagePrinter = Mockito.mock(MessagePrinter.class);
+    TicketStorage ticketStorage = new TicketStorage();
+    TicketPrinter ticketPrinter = new TicketPrinter(messagePrinter);
+
+    @BeforeEach
+    void putTicketsIntoTicketStorage() {
+        for (Ticket ticket : Ticket.values()) {
+            ticketStorage.addTicket(ticket);
+        }
+    }
+
+    @Test
+    void shouldPrintAllTicketsFromTicketStorage() {
+
+//        when
+        ticketPrinter.printTicketsFromTicketStorage(ticketStorage);
+
+
+//        then
+        then(messagePrinter)
```

Comment:
add assertion that `ticketStorage` was cleared

Reply:
Done